sources/vega/vega.proto

```diff
@@ -1146,7 +1146,7 @@ message Delegation {
 }
 
 // Details for a single reward payment
-message RewardDetails {
+message Reward {
```

Comment:
These are fine by me too, but this is a breaking change with the core and will need to be changed there as well.

Reply:
What do you think it will it break? As far as I can figure it is only used in the Data Node. In the core they get put on the event bus as 'RewardPayout' events (yay another name for the same thing :)

```
➜  vega git:(develop) grep -r RewardDetails *
➜  vega git:(develop) grep -r RewardPer *   
```